test/recipes/95-test_external_gost_engine_data/gost_engine.sh

```diff
@@ -37,9 +44,11 @@ cd $SRCTOP/gost-engine
 rm -rf build
 mkdir -p build
 cd build
-cmake ..
+cmake .. -DOPENSSL_ROOT_DIR="$OPENSSL_ROOT_DIR"
```

Comment:
Argh!  No!  Don't build in the source tree!

As a matter of fact, there's no need to move anywhere, as this script should be standing in `test-runs/test_external_gost_engine` already.  So, the bunch of lines from `cd $SRCTOP/gost-engine` to here can be replaced with this:

``` bash
cmake $SRCTOP/gost-engine -DOPENSSL_ROOT_DIR="$OPENSSL_ROOT_DIR"
```

Reply:
Yeah, I just was not changing it from what was there before already. But I can try this.

Reply:
> Argh! No! Don't build in the source tree!
> 
> As a matter of fact, there's no need to move anywhere, as this script should be standing in `test-runs/test_external_gost_engine` already. So, the bunch of lines from `cd $SRCTOP/gost-engine` to here can be replaced with this:
> 
> ```shell
> cmake $SRCTOP/gost-engine -DOPENSSL_ROOT_DIR="$OPENSSL_ROOT_DIR"
> ```

I'm sorry. I partially look at all this stuff as at Ancient Magic, and prefer not to dive in it...